src/main/java/org/jmhsrobotics/frc2023/Robot.java

```diff
@@ -73,7 +73,7 @@ public void robotPeriodic() {
 		// and running subsystem periodic() methods. This must be called from the
 		// robot's periodic
 		// block in order for anything in the Command-based framework to work.
-		CommandScheduler.getInstance().run();
+		CommandScheduler.getInstance().run(); // TODO: delete
 		field.setRobotPose(m_robotContainer.drivetrain.getPose());
```

Comment:
What is this line for?

Reply:
oh shoot sorry I completely forgot that I wrote that, I just thought it was funny